skrub/_total_std_norm.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import sklearn
+from sklearn.utils.fixes import parse_version
+from sklearn.utils.validation import check_array
+
+from . import _dataframe as sbd
+
+
+def total_std_norm(X):
```

Comment:
what does std stand for

Reply:
I need to update the docstring below slightly